src/clanhr/auth/authorization_rules.clj

```diff
@@ -5,7 +5,8 @@
   "Template role profiles for easy access"
   {:full-access ["admin" "hrmanager" "manager" "user" "" "staff"]
    :board-member-manager ["admin" "hrmanager" "manager" "staff"]
-   :board-member ["admin" "hrmanager" "staff"]})
+   :board-member ["admin" "hrmanager" "staff"]
```

Comment:
Hum... @lamuria I think the `staff` was something like support/product people, that could see hidden things.